components/_expanding-components/uncheck-radio-button.js

```diff
@@ -0,0 +1,37 @@
+export const UncheckRadioButton = function(elementClass) {
+  const expandingComponent = elementClass
+  let expandingElement
+
+  const cacheDom = function() {
+    expandingElement = document.querySelectorAll('.' + expandingComponent)
+  }
+
+  const bindEvents = function() {
+    expandingElement.forEach(labelElement => {
+      labelElement.addEventListener('click', () => findInputForLabel(labelElement))
+    })
+  }
+
+  const findInputForLabel = function(labelElement) {
```

Comment:
findInputForLabel name suggests that this function finds and probably returns an input using label as identifier. The final result of the function is different - it sets checked attribute to false of input with specified label (label as argument). I would suggest to change name of the function in order to reflect what this funtion does.